XenAdmin/Commands/BugToolCommand.cs

```diff
@@ -31,6 +31,7 @@
 
 using XenAdmin.Network;
 using XenAdmin.Wizards;
+using XenAdmin.Wizards.BugToolWizardFiles;

```

Comment:
It looks like this is an unused import.

Reply:
well spotted. Also, there's quite a few, better to get rid of all of them 😎